src/pages/users/users-details/users-details.js

```diff
@@ -0,0 +1,222 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector, useDispatch } from 'react-redux';
+import {
+  FormControl,
+  Grid,
+  Paper,
+  TextField,
+  Avatar,
+  Typography,
+  Button
+} from '@material-ui/core';
+import { withRouter } from 'react-router';
+import { useStyles } from './users-details.styles';
+import useUsersHandler from '../../../utils/use-users-handlers';
+import LoadingBar from '../../../components/loading-bar';
+import { config } from '../../../configs';
+import { updateUserStatus } from '../../../redux/users/users.actions';
+import { closeDialog } from '../../../redux/dialog-window/dialog-window.actions';
+import useSuccessSnackbar from '../../../utils/use-success-snackbar';
+
+const {
+  USER_ACTIVE_TITLE,
+  USER_UNACTIVE_TITLE,
+  SWITCH_USER_STATUS_TITLE
+} = config.buttonTitles;
+const { USER_ACTIVE_STATUS, USER_UNACTIVE_STATUS } = config.statuses;
+const { SWITCH_USER_STATUS_MESSAGE } = config.messages;
+
+const UsersDetails = (props) => {
+  const { match } = props;
+
+  const { openSuccessSnackbar } = useSuccessSnackbar();
+  const styles = useStyles();
+  const dispatch = useDispatch();
+
+  const { loading } = useSelector(({ Users }) => ({
+    loading: Users.usersLoading
+  }));
+
+  const { id } = match.params;
+
+  const {
+    firstName,
+    lastName,
+    country,
+    city,
+    adress,
+    postCode,
+    ban
+  } = useUsersHandler(id);
+
+  const status = ban ? USER_UNACTIVE_STATUS : USER_ACTIVE_STATUS;
+  const buttonStatus = ban ? USER_ACTIVE_TITLE : USER_UNACTIVE_TITLE;
+
+  const userStatusHandler = (userId) => {
+    const updateStatus = () => {
+      dispatch(closeDialog());
+      dispatch(updateUserStatus(userId));
+    };
+    openSuccessSnackbar(
+      updateStatus,
+      SWITCH_USER_STATUS_TITLE,
+      SWITCH_USER_STATUS_MESSAGE,
+      SWITCH_USER_STATUS_TITLE
+    );
+  };
+
+  if (loading) {
+    return <LoadingBar />;
+  }
+
+  return (
+    <div className={styles.detailsContainer}>
+      <form>
```

Comment:
Why do we need a tag form here if we don't use it at all?

Reply:
fixed. sry , my bad